Jellyfin.Drawing.Skia/SkiaEncoder.cs

```diff
@@ -186,6 +205,9 @@ private SKBitmap CropWhiteSpace(SKBitmap bitmap)
         }
 
         /// <inheritdoc />
+        /// <exception cref="ArgumentNullException">If path is null.</exception>
+        /// <exception cref="FileNotFoundException">If the path is not valid.</exception>
+        /// <exception cref="SkiaCodecException">If the file at the specified path could not be used to generate a codec.</exception>
```

Comment:
Minor comment, drop the "If" at the beginning, it's more inline with other exception docs.
e.g. https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpclient.sendasync?view=netframework-4.7#System_Net_Http_HttpClient_SendAsync_System_Net_Http_HttpRequestMessage_System_Threading_CancellationToken_

Reply:
Addressed in  f7eef1a